osu.Game/Screens/Ranking/Statistics/StatisticsPanel.cs

```diff
@@ -75,7 +75,21 @@ private void populateStatistics(ValueChangedEvent<ScoreInfo> score)
                 return;
 
             if (newScore.HitEvents == null || newScore.HitEvents.Count == 0)
-                content.Add(new MessagePlaceholder("Score has no statistics :("));
```

Comment:
Please wrap this in a `{ }`

```
if (...)
    x.Add(
    {
        ...
    }
else
{
...
}
```
Is almost bound to lead to errors.

Reply:
you mean just adding a pair of `{}` for the outer if, correct?

Reply:
yes

Reply:
it's actually enforced by our style guidelines, overlooked.